src/cpp/src/model_runner.hpp

```diff
@@ -161,7 +161,7 @@ class ModelRunner {
         int64_t *input_ids_data = nullptr;
         
         if (sequence_group_type == SequenceGroupType::EMBEDDINGS) {
-            OPENVINO_ASSERT(m_embedding.get_request(), "Got sequence group with embeddings, but embeddings model wasn't set.");
+            // OPENVINO_ASSERT(m_embedding.get_request(), "Got sequence group with embeddings, but embeddings model wasn't set.");
```

Comment:
```suggestion
```
There's another commented out assert in src/cpp/src/visual_language/embedding_model.cpp